trillian.proto

```diff
@@ -139,9 +139,6 @@ message Tree {
   // This can be any type of message to accommodate different key management
   // systems, e.g. PEM files, HSMs, etc.
   // Private keys are write-only: they're never returned by RPCs.
```

Comment:
Could you document the update logic? Ie, that it's valid to change providers, but the key itself can't change.

Reply:
Done.